SingularityBase/src/main/java/com/hubspot/singularity/SingularityState.java

```diff
@@ -85,6 +86,7 @@ public SingularityState(@JsonProperty("activeTasks") int activeTasks,
                           @JsonProperty("oldestDeployStep") long oldestDeployStep,
                           @JsonProperty("activeDeploys") List<SingularityDeployMarker> activeDeploys,
                           @JsonProperty("lateTasks") int lateTasks,
+                          @JsonProperty("listLateTasks") List<SingularityPendingTask> listLateTasks,
```

Comment:
we probably just want to use the SingularityPendingTaskId here. That will give us the basic information about the request/deploy it belongs to. The SingularityPendingTask can be a very large amount of data (includes lists of artifacts, environment variables, etc, etc), which we wouldn't want to be sending on every call for the state